cumulusci.yml

```diff
@@ -290,18 +299,26 @@ flows:
                 options:
                     unmanaged: False
 
-    install_regression:
+    config_regression:
         steps:
-            # Add QA metadata needed for regression
-            5:
+            1:
                 task: deploy_qa_config
                 options:
                     unmanaged: False
-            6:
+            2:
                 task: update_admin_profile
                 options:
                     managed: true
 
+    config_trial:
+        steps:
+            1:
+                flow: config_qa
```

Comment:
At some point we probably want to refactor it so that you can get the trial-like configuration with or without the additional QA-specific metadata from unpackaged/config/qa